packages/docs/blocks/input/CheckboxSwitch.yaml

```diff
@@ -0,0 +1,26 @@
+# Copyright 2020-2021 Lowdefy, Inc
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+_ref:
+  path: templates/blocks/template.yaml.njk
+  vars:
+    block_type: CheckboxSwitch
+    value_type: any[]
+    category: input
+    schema: ../blocks/blocksAntd/src/blocks/CheckboxSwitch/CheckboxSwitch.json
+    filePath: blocks/input/CheckboxSwitch.yaml
+    description_content: |
+      The `CheckboxSwitch` block allows a user to select a boolean value between (true/false).
+
+      > A similar switch block is `Switch`.
```

Comment:
Please add something about the CheckboxSelector as well, in case users are confused between the two

Reply:
Note added 👍